src/system_info/system_info_display_wayland.cc

```diff
@@ -204,7 +204,7 @@ bool SysInfoDisplay::UpdateBrightness() {
   int val = atoi(str_val);
   free(str_val);
 
-  brightness_ = val / max_val;
```

Comment:
My only concern has always been that these are two separate fixes that could be committed separately (even if part of the same pull request). And if both `max_val` and `val` were `double`s (and then we use `strtod()` instead of `atoi()`) we could avoid the cast altogether.